packages/cms-editor/src/main/MainContent.tsx

```diff
@@ -62,17 +63,19 @@ export const MainContent = () => {
       minSize: 200,
       size: 500,
       maxSize: 1000
-    },
-    {
-      id: defaultLanguageTag,
-      accessorFn: co => co.values[defaultLanguageTag],
-      header: ({ column }) => <SortableHeader column={column} name={languageDisplayName.of(defaultLanguageTag) ?? defaultLanguageTag} />,
+    }
+  ];
+  toLanguages(defaultLanguageTags, languageDisplayName).forEach(language =>
```

Comment:
maybe it would be a good idea to wrap this column into a useMemo, as this only needs to change if this languages changes